docs/API/plain_rules.md

```diff
@@ -117,7 +117,10 @@ You should take into consideration the following guidelines:
 -   Metadata is also available as explained in the [metadata and object values](#metadata-and-object-values) section.
 -   The variable `ruleName` in automatically added to the action, even if it is not present in the EPL text. The
     ruleName automatically added this way is retrieved as part of the EPL text when the rule is recovered using
-    `GET /rules` or `GET /rules/{name}`.
+    `GET /rules` or `GET /rules/{name}`. For time based rules if `rulename` was not provided in EPL text then will be
+    added automatically by using an unique format, which inludes service and subservice (i.e. provided name rule was
+    `myrule` in service `smartcity` and subservice `/gerdens` then ruleName for EPL text will be
+    `myrule@smartcity/gardens`)
```

Comment:
```suggestion
    `GET /rules` or `GET /rules/{name}`. For time based rules if `ruleName` was not provided in EPL text, then it will be
    generated automatically ensuring uniqueness, by including service and subservice in such name (i.e. provided name 
    rule was `myrule` in service `smartcity` and subservice `/gardens` then ruleName for EPL text will be
    `myrule@smartcity/gardens`)
```

Reply:
finally was discarded that approach